### ISSUE TITLE
Nav A11y Pass

### ISSUE THREAD

**Issue by TestAccessibilityGIT** [OfficeDev/office-ui-fabric-react#7803]
Tracking all of the issues brought up with the 1/25 accessibility pass on Nav control 
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B8b59713f-87bb-4a5e-ad92-2b7b3a0ef589%7D&action=default&uid=%7B8B59713F-87BB-4A5E-AD92-2B7B3A0EF589%7D&ListItemId=1721&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble) 
  
 - [ ] Bug1: [Accessibility][Keyboard navigation]: Landmarks must have a unique role or role/label      combination. 
- [ ]  Bug2: [Accessibility][Visual requirements]: Elements must have sufficient color contrast." 
- [ ]  Bug3: [Accessibility][Programmatic access]: Name of the 'Basic components expand button' is not descriptive to the user.  
- [ ]  Bug4: [Accessibility][Programmatic access]: Selected state is not defined for controls under 'Nav bar of links each with unique keys and empty URLs’ section.  
- [ ]  Bug5: [Accessibility][Keyboard navigation]: Tooltip does not appear on keyboard focus.  
- [ ]  Bug6: [Accessibility][Supporting the platform]: Link controls adopts disable text color in  high contrast black/white mode.  
- [ ]  Bug7: [Accessibility][Supporting the platform]: Keyboard focus is not visible in high contrast black/white theme.   
- [ ]  Bug8: [Accessibility][Programmatic access]: User does not get sense of hierarchy and number of child elements while navigating 'Nested navigation menu'.  
 
 
 Component: Nav 
@micahgodbolt

**Comment by HamletDRC** [comment]
My product at Microsoft has a MAS C violation because of Bug #8. Can you possible provide a delivery date on this? My SLA to fix this issue is 1. October, so I really need this one fixed in September or we're going to have to implement our own menu.

**Comment by micahgodbolt** [comment]
taking my response offline

**Comment by jdhuntington** [comment]
Breaking issues out into discrete items.
